llama_index/llms/base.py

```diff
@@ -50,7 +50,15 @@ def __str__(self) -> str:
 
 # ===== Generic Model Output - Completion =====
 class CompletionResponse(BaseModel):
-    """Completion response."""
+    """
+    Completion response.
+
+    Fields:
+        text: TODOC.
+        additional_kwargs: TODOC.
```

Comment:
@logan-markewich can you suggest to me what to put for these?

Reply:
text -- the text content of the response (or the current state of the full text, if streaming)

additional_kwargs -- additional information on the response i.e. token counts, function calling information

Reply:
Thank you! 👍